public/main/inc/lib/api.lib.php

```diff
@@ -2712,10 +2712,16 @@ function api_get_setting($variable, $isArray = false, $key = null)
             if (in_array($environment, $test)) {
```

Comment:
I think that logic needs to change, because if the environment (I suppose from APP_ENV in .env) is "dev", then `server_type` can never be "prod".

Reply:
Confirmed. Please ignore the $environment to set the current server_type.